atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/impl/TableRemappingKeyValueServiceTest.java

```diff
@@ -89,6 +92,67 @@ public void canConcurrentlyReadFromStableTablesWhileCreatingAndDroppingOtherTabl
         }
     }
 
+    @Test
+    public void concurrentlyCreateAndDeleteTablesAcrossManyWriters() {
+        int maxWritersMaybeWeAreSatisfiedTheBugIsGone = 2;
```

Comment:
Even at 2 writers this can fail. Setting to more than about 3 seems to cause a failure 100% of the time, at least locally.